packages-ts/gauntlet-terra-cw-plus/src/commands/multisig.ts

```diff
@@ -91,15 +94,38 @@ export const wrapCommand = (command) => {
       }
     }
 
+    parseVotingPeriod(votingPeriod: string): number {
+      try {
+        Number(votingPeriod)
```

Comment:
This will never throw. You could add a `isNaN` check after it. Something like:
```js
const n = Number(votingPeriod)
if (isNaN(n)) throw new Error(...)
...
```